src/migrations/20240208123212-create-stat-traffic-usage-table.js

```diff
@@ -0,0 +1,24 @@
+
+exports.up = (db, callback) => {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS stat_traffic_usage(
+            day DATE NOT NULL,
+            traffic_group TEXT NOT NULL,
+            status_code_series INT NOT NULL,
```

Comment:
appended a _series to make it clear that it's 200/300 etc, not 200/201/202/304 etc